src/index.ts

```diff
@@ -6,3 +6,6 @@ export * from "./api";
 export * from "./client";
 export * from "./utils/apiEndpoints";
 export * from "./bcs";
+export * from "./types";
```

Comment:
we need to export only `types/index` and `type/indexer`. exporting everything will get us with a huge package size, and that is the whole point in how we structure indexer type

Reply:
Done